coreos-koji-tagger/coreos_koji_tagger.py

```diff
@@ -316,6 +316,12 @@ def process_lockfiles(self, commit=None):
                     else:
                         # Log any errors we encounter. 404s are ok, but won't hurt to log
                         logger.warning('URL request error: %s' % r.text.strip())
+        if not desiredrpms:
+            logger.warning('We detected no desiredrpms.')
+            logger.warning("Does the repo/ref: %s/%s have any lockfiles?" %
+                            (self.github_repo_branch, self.github_repo_fullname))
```

Comment:
Those args look swapped? (Also, why not use f-strings? I find them much nicer to write/read!)

Reply:
Sorry copy/paste error.

I like f strings too, but sometimes the lines get too long and breaking them up sucks too.